pkg/controller/install/install_controller.go

```diff
@@ -166,9 +169,15 @@ func (r *ReconcileInstall) install(instance *tektonv1alpha1.Install) error {
 	filters := []mf.FilterFn{
 		mf.ByOwner(instance),
 	}
-
 	r.manifest.Filter(filters...)
-	return r.manifest.ApplyAll()
+	rc := r.manifest.ApplyAll()
+
+	for _, path := range instance.Spec.AddOns {
+		extension := mf.NewYamlManifest(filepath.Join("deploy", "resources", path), true, r.config)
+		extension.Filter(filters...)
+		extension.ApplyAll()
+	}
```

Comment:
Will the addon resources be deleted when we remove the instance of `install` CRD. Resources in `r.manifest` are deleted (line 138) when the CRD instance is removed.

Reply:
Yes, they will.  The addon resources are owned by the install instance.  When the install instance is deleted, the addon resources owned by the instance are deleted automatically.